app/content/models/strike.py

```diff
@@ -0,0 +1,42 @@
+import uuid
+
+from django.db import models
+
+from app.content.models import Event, User
+from app.util.models import BaseModel
+from app.util.utils import today
+
+
+class Strike(BaseModel):
+    id = models.UUIDField(
+        auto_created=True, primary_key=True, default=uuid.uuid4, serialize=False,
+    )
+    description = models.CharField(max_length=200)
+    expires_at = models.DateTimeField()
+    nr_of_strikes = models.IntegerField(default=1)
```

Comment:
hva er denne til, hvilket strike dette er eller?

Reply:
nei er antall strikes. Om en person får 3 prikker for å ikke møte opp vi vi ikke lage tre objekter, men heller summere opp eksisterende